.gitignore

```diff
@@ -15,3 +15,8 @@ __pycache__
 .venv
 .mypy_cache
 
+# dbt
+profiles.yml
+airbyte-integrations/bases/base-normalization/dbt-project-template/logs/
```

Comment:
Should these be in `airbyte-integrations/bases/base-normalization/dbt-project-template/.gitignore`?

Or maybe we could exclude everything like https://github.com/fishtown-analytics/dbt-utils/blob/master/.gitignore:
```
logs
models/generated
dbt_modules
```